server/sonar-server/src/test/java/org/sonar/server/qualityprofile/ws/QProfilesWsTest.java

```diff
@@ -172,7 +172,7 @@ public void define_set_default_action() {
     WebService.Action setDefault = controller.action("set_default");
     assertThat(setDefault).isNotNull();
     assertThat(setDefault.isPost()).isTrue();
-    assertThat(setDefault.params()).hasSize(3);
+    assertThat(setDefault.params()).hasSize(4);
```

Comment:
No change required anymore, I removed the new organization parameter (see comment below).